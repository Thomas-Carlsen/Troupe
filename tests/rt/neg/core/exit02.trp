exit (attenuate (authority, {alice}), 0)
