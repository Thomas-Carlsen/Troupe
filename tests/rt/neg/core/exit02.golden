2019-03-07T16:06:58.364Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
[31mRuntime error in thread cefa1106-2e46-4b7d-b0e9-5cfc6350710b@{}%{}[39m
[31m>> Provided authority is not TOP[39m
[31m | level of the provided authority: {alice}[39m
